applications/FluidDynamicsApplication/tests/two_fluid_hydraulic_test.py

```diff
@@ -0,0 +1,200 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+import math
+import time
+import sys
+
+import KratosMultiphysics.KratosUnittest as UnitTest
+import KratosMultiphysics.kratos_utilities as KratosUtilities
+
+class FluidDynamicsAnalysisHydraulic(FluidDynamicsAnalysis):
+
+    def __init__(self,model,project_parameters,flush_frequency=10.0):
+        super().__init__(model,project_parameters)
+
+        self.flush_frequency = flush_frequency
+        self.last_flush = time.time()
+
+    def ModifyInitialGeometry(self):
+
+            h0=0.5
+            hmax=0.1
+            for node in self._GetSolver().GetComputingModelPart().Nodes:
+                d0=node.Y-(h0+hmax*math.cos(math.pi*(1-node.X)))
+                node.SetSolutionStepValue(KratosMultiphysics.DISTANCE,d0)
+                node.SetSolutionStepValue(KratosMultiphysics.DISTANCE,1,d0)
+
+    def FinalizeSolutionStep(self):
+        super().FinalizeSolutionStep()
+        if self.parallel_type == "OpenMP":
+            now = time.time()
+            if now - self.last_flush > self.flush_frequency:
+                sys.stdout.flush()
+                self.last_flush = now
+
+# Class derived from the UnitTest (KratosMultiphysics.KratosUnittest) class
+class TwoFluidHydraulicSolverTest(UnitTest.TestCase):
+
+    def setUp(self):
+        self.work_folder = "TwoFluidMassHydraulicTest"
```

Comment:
```suggestion
        self.work_folder = "two_fluid_hydraulic_solver_test"
```
Please follow filename convention.

Reply:
Besides, I think that you're testing more stuff than the mass.